libraries/module/lib/util/Configuration.ts

```diff
@@ -16,25 +16,64 @@
  * limitations under the License.
  */
 import Yargs = require("yargs");
+import { LoggingOptions } from "@syntest/logging";
+
+export type GeneralOptions = {
```

Comment:
The storage options make sense to move as the storage might change depending on the modules. However, it is a bit odd that the config type is in module but the parameter is in cli. We should maybe move the cli arg to the module as well. Does the cli have to define this?

Reply:
Not necessarily. It just felt weird to have the usage of the tool defined in the module package. But I moved it now. The CLI now only contains the bin.ts file.